foundationsync/indico_foundationsync.py

```diff
@@ -231,8 +239,17 @@ def fetch_rooms(self, connection, room_name=None):
 
             # Update room data
             room = self._update_room(room, room_data, room_attrs)
+            for manager in room.get_manager_list():
+                room.update_principal(manager, full_access=False)
+            if room_data.get('owner') is not None:
+                room.update_principal(room_data['owner'], full_access=True)
+            manager_group = room_attrs.get('manager-group')
+            if manager_group is not None:
+                group = GroupProxy.get_named_default_group(manager_group)
```

Comment:
The docstring of this method says it's deprecated.

Reply:
Add a plugin setting to select the identity provider used (since gesloc just gives us group names) or hardcode `cern-ldap` for the provider name (setting is cleaner)

Reply:
I'd just hardcode it. It's a CERN plugin anyway. You cannot be more CERN than Foundation.